### PR TITLE
Update version strings to 8.2.0.b1 for beta release.

### LINKED ISSUE #490: Update version strings to 8.2.0.b1 for beta release.
Update documentation to reflect new release.

### PR DESCRIPTION
**Pull request by mattgreene**
Issues:
Fixes #490

Problem:
Documentation needs to reflect latest version.

Analysis:
Search and replace 8.1.0 with 8.2.0.b1.
Update release notes file present in repo.

Tests:
Review docs in fork prior to submitting PR to F5Networks.

### CODE REVIEW

**Comment by richbrowne** [comment]
Matt,

  You are going to hate me, but I think that using the extra .b1 will break the python package build.  I can't remember what broke, but there was a reason why the alpha versions were a1 and a2 and we didn't use the dot.

I am going to play around with this.

**Comment by jputrino** [comment]
Yes, the doc link should point to lbaasv2-driver.

-- Jodie Putrino

On Dec 16, 2016, at 7:48 PM, Jeffrey Longstaff <notifications@github.com<mailto:notifications@github.com>> wrote:


@jlongstaf commented on this pull request.

________________________________

In docs/release_notes.rst<https://github.com/F5Networks/f5-openstack-agent/pull/491#pullrequestreview-13439340>:

>  Open Issues
 -----------

 See the `project issues page <https://github.com/F5Networks/f5-openstack-agent/issues>`_ for a full list of open issues in this release.

-Documentation can be found on `Read the Docs <http://f5-openstack-lbaasv2-driver.readthedocs.io/en/latest/release_notes.html>`_.
+Documentation can be found on `Read the Docs <http://f5-openstack-agent.readthedocs.io/en/latest/release_notes.html>`_.


This is a broken link. I thought release notes were always in the driver repo. @jputrino<https://github.com/jputrino> would know for sure.

-

**Comment by richbrowne** [comment]
Matt,

The versions of the dependencies need to be updated.  For example, the new sdk dependency is on version 2.1.0.  That release depends on icontrol-rest version v1.1.0

     $ curl -L -O https://github.com/F5Networks/f5-common-python/releases/download/v1.5.0/f5-sdk-1.5.0-1.el7.noarch.rpm
     $ curl -L -O https://github.com/F5Networks/f5-icontrol-rest-python/releases/download/v1.0.9/f5-icontrol-rest-1.0.9-1.el7.noarch.rpm
     $ sudo rpm –ivh f5-icontrol-rest-1.0.9-1.el7.noarch.rpm f5-sdk-1.5.0-1.el7.noarch.rpm

File: docs/release_notes.rst [hunk 1]
```diff
@@ -11,29 +11,20 @@ Release Highlights
...
-* L7 routing
-* Loadbalancer statistics
-
 Open Issues
 -----------
 
 See the `project issues page <https://github.com/F5Networks/f5-openstack-agent/issues>`_ for a full list of open issues in this release.
 
-Documentation can be found on `Read the Docs <http://f5-openstack-lbaasv2-driver.readthedocs.io/en/latest/release_notes.html>`_.
+Documentation can be found on `Read the Docs <http://f5-openstack-agent.readthedocs.io/en/latest/release_notes.html>`_.
```
Comment (mattgreene) [inline docs/release_notes.rst]: The link works for me.  It needs to point to the corresponding project for the thing being released.  In the future, we may unbundle the driver and agent w.r. versions.

**Comment by mattgreene** [comment]
Fixed Rich's comment on sdk and icontrol versions.

Re: Jeff's comment.  The agent and driver packages will eventually need to move independently from each other.  We either need to remove the release_notes file from agent, or keep what I have now (which updated the file in both repo to reflect the same information, but with appropriate links).

**Comment by jputrino** [comment]
I'm fine with having it in both.

-- Jodie Putrino

On Dec 17, 2016, at 10:49 AM, Matt Greene <notifications@github.com<mailto:notifications@github.com>> wrote:


Fixed Rich's comment on sdk and icontrol versions.

Re: Jeff's comment. The agent and driver packages will eventually need to move independently from each other. We either need to remove the release_notes file from agent, or keep what I have now (which updated the file in both repo to reflect the same information, but with appropriate links).

-

**Pull request merged**